python/tk_desktop/project_model.py

```diff
@@ -19,6 +19,7 @@
 
 shotgun_model = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_model")
 shotgun_login = sgtk.platform.import_framework("tk-framework-login", "shotgun_login")
+shotgun_api3 = sgtk.platform.import_framework("tk-framework-login", "shotgun_api3")
```

Comment:
This seems like it should be

```python
from tank_vendor import shotgun_api3
```

Simpler and we're getting out of the business of the login framework handling connections, which this feels related to.